flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/data/FlinkParquetReaders.java

```diff
@@ -34,7 +34,6 @@
 import org.apache.flink.table.data.RowData;
```

Comment:
The PR needs to go against the 2.1 target (currently: 2.0), as #13714 has been merged.